scalding-base/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -40,6 +38,15 @@ import scala.concurrent.{
 import scala.util.{Failure, Success, Try}
 import scala.util.hashing.MurmurHash3
 
+trait Config {
```

Comment:
this needs to be built up a bit more and there is a duplicated class with the same name in core.